<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

  
<!--   <include file="$(find openni2_launch)/launch/openni2.launch"/> -->
<!--   <arg name="robotID" default="44" /> -->
  <arg name="robotID" default=""/>
  <arg name="mateID" default=""/>
 
  <param name="selfDetectionTopicName" value="tgt_detector$(arg robotID)/detectedObject_Wframe_byRobot$(arg robotID)" type="string" />    
  <param name="mateDetectionTopicName" value="tgt_detector$(arg mateID)/detectedObject_Wframe_byRobot$(arg mateID)" type="string" />  
  <param name="fusedMeasurementTopicName" value="fusedObject_Wframe_byRobot$(arg robotID)"/> type="string"
  
  <node name="tgt_fusor$(arg robotID)" pkg="tgt_measurement_fusion" type="tgt_fusor" output="screen">
    
  </node>

</launch>